magicctapipe/scripts/lst1_magic/lst1_magic_mc_dl0_to_dl1.py

```diff
@@ -420,6 +475,15 @@ def mc_dl0_to_dl1(input_file, output_dir, config):
     logger.info('\nOutput file:')
     logger.info(output_file)
 
+    if muons_analysis:
+        dir, name = os.path.split(output_file)
+        name = name.replace('dl1', 'muons').replace('LST-1.1', 'LST-1')
```

Comment:
It seems this is a copy from lstchain, but the second replacement assumes that the input is a LST-1 real data file. It would be good to remove it to avoid a confusion.